README.md

```diff
@@ -16,59 +16,76 @@ dotnet add package Serilog.AspNetCore
 
 **Next**, in your application's _Program.cs_ file, configure Serilog first.  A `try`/`catch` block will ensure any configuration issues are appropriately logged:
 
+_ASP.Net 6 combines the `Startup` and `Program.Main`._
+
+Update your `Program.cs` file like this
+
 ```csharp
 using Serilog;
 using Serilog.Events;
 
-public class Program
+var builder = WebApplication.CreateBuilder(args);
+
+//1. Add logger setup
+Log.Logger = new LoggerConfiguration()
+    .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+    .Enrich.FromLogContext()
+    .WriteTo.Console()
```

Comment:
This line is a bit problematic.

Printing to the console is slow. Like 100x slowdown in some of my tests.

In my code I have wrapped it in a `#IF DEBUG` block
I was wondering if the was a better (less ugly) way we should be disabling this in prod; and including that in the example given?

Reply:
Might be a good one to chat about in another thread :-)

Reply:
I'll create an issue to discuss this in

Reply:
#283